src/main/java/edu/ucla/library/iiif/fester/verticles/V2ManifestVerticle.java

```diff
@@ -609,6 +609,10 @@ private Metadata updateMetadata(final Metadata aMetadata, final String... aStrin
                     metadata.add(entry);
                 } // If label matches but we have no updated value, ignore the metadata
             }
+        } else {
+            for (int index = 0; index < aStringArray.length / 2; index += 2) {
```

Comment:
Not saying this is wrong, but would like an explanation for the `aStringArray.length / 2` and i`ndex += 2` values in the loop.